openfisca_uk/variables/government/dwp/pension_credit/eligible.py

```diff
@@ -0,0 +1,16 @@
+from openfisca_uk.model_api import *
+
+
+class pension_credit_eligible(Variable):
```

Comment:
I think our standard is to include verbs in bools, could you check elsewhere?
```suggestion
class is_pension_credit_eligible(Variable):
```